assets/javascript/game2.js

```diff
@@ -1,141 +1,69 @@
 $( document ).ready(function() {
     //console.log( "ready!" );
 
-//global variables
-  let wins = 0;
-  let losses = 0;
-  let numberToGuess;
-  let playerTotal = 0;
-  let value;
+    //global variables
+    const reset = init;
+    let wins = 0;
+    let losses = 0;
+    let numberToGuess;
+    let playerTotal;
 
-  
-function reset (){
-    $(".crystal").each(function(){
-        numberToGuess = Math.floor(Math.random() * 101 + 19);
-        //console.log(value);
-        {
```

Comment:
no reason for a block here (removing the curlies on lines 16 and 20 would make it more readable and behave the same way)